core/common/src/main/java/alluxio/Configuration.java

```diff
@@ -55,47 +54,69 @@
  * distributed with Alluxio jar. Alluxio users can override values of these default properties by
  * creating {@code alluxio-site.properties} and putting it under java {@code CLASSPATH} when running
  * Alluxio (e.g., ${ALLUXIO_HOME}/conf/)
- *
- * <p>
- * Developers can create an instance of this class by {@link #Configuration()}, which will load
- * values from any Java system properties set as well.
- *
- * <p>
- * The class only supports creation using {@link #Configuration(Properties)} to override default
- * values.
  */
 @NotThreadSafe
 public final class Configuration {
   /** File to set default properties. */
   public static final String DEFAULT_PROPERTIES = "alluxio-default.properties";
-  /** File to set customized properties. */
+  /** File to set customized properties for Alluxio deployment. */
```

Comment:
Does this mean both client and server? If so I think its clearer to just state it explicitly.